File: iniciante/1009/1009.cs
using System; 

class URI {

    static void Main(string[] args) { 

        /*
        Faça um programa que leia o nome de um vendedor, o seu salário fixo e o total de vendas efetuadas por ele no mês (em dinheiro). Sabendo que este vendedor ganha 15% de comissão sobre suas vendas efetuadas, informar o total a receber no final do mês, com duas casas decimais.
         */
        float comissao = 0.15f;

        string vendedor = Console.ReadLine();
        double fixo = float.Parse(Console.ReadLine());
        double vendas = float.Parse(Console.ReadLine());

        double total = fixo + vendas * comissao;

        Console.WriteLine("TOTAL = R$ {0:F2}", total);
        Console.ReadLine();
    }

}

File: iniciante/1009/1009.go
package main

import (
	"fmt"
)

func main() {
	var nome string
	var sal, vendas float64

	fmt.Scan(&nome)
	fmt.Scan(&sal)
	fmt.Scan(&vendas)

	total := sal + vendas*.15

	fmt.Printf("TOTAL = R$ %.2f\n", total)
}
